REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,334 @@
+replication is complete and also allows the rate of any periodic replication required by the application to be varied
+depending on whether there are components bound to the service or not. For example, it may be desirable to have an app that
+triggers replications every 24 hours in the background when the app is not actively being used, but when the app is being
+used and a particular [`Activity`](http://developer.android.com/reference/android/app/Activity.html)
+is displayed it may be desirable to replicate every few minutes and then update the
+[`Activity`](http://developer.android.com/reference/android/app/Activity.html) to show the data when replication has completed.
+
+#### ReplicationService
+
+`ReplicationService` is an abstract class and your subclass must implement the following abstract method:
+* `protected abstract Replicator[] getReplicators(Context context)` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+
+#### PeriodicReplicationService
+
+This abstract class is a child of `ReplicationService` and requires you to implement the following abstract methods:
+* `protected abstract Replicator[] getReplicators(Context context)` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+* `protected abstract int getBoundIntervalInSeconds()` This method should return the interval (in seconds) you wish to have between replications when components are bound to the Service.
+* `protected abstract int getUnboundIntervalInSeconds()` This method should return the interval (in seconds) you wish to have between replications when components are not bound to the Service.
+* `protected abstract boolean startReplicationOnBind()` This should return `true` if you wish to have replications triggered immediately when a component binds to the Service.
+
+Note that internally this uses [android.app.AlarmManager.setInexactRepeating()](http://developer.android.com/reference/android/app/AlarmManager.html#setInexactRepeating(int, long, long, android.app.PendingIntent)) 
+to schedule the repetition of the replications at the given intervals in a battery efficient way. This means the intervals between replications will not be exact.
+
+#### PeriodicReplicationReceiver
+
+`PeriodicReplicationReceiver` uses a 
+[`WakefulBroadcastReceiver`](http://developer.android.com/reference/android/support/v4/content/WakefulBroadcastReceiver.html)
+to trigger periodic replications at the intervals specified by your
+`PeriodicReplicationService`. This means your application does not have to keep running to trigger replications at the
+intervals you require, but will be restarted at the time a replication is required. This class also handles resetting of
+the periodic replications after a reboot of the device. You must add the following `intent-filter`s to your
```

Comment:
This should say `<action> elements to the <intent-filter> of the PeriodicReplicationReceiver's entry in the 
AndroidManifest.xml.